policies/templates/gcp_gke_disable_legacy_endpoints_v1.yaml

```diff
@@ -0,0 +1,80 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Check to see if GKE legacy endpoints is disabled.
+# https://cloud.google.com/kubernetes-engine/docs/how-to/hardening-your-cluster#protect_node_metadata
+
+apiVersion: templates.gatekeeper.sh/v1alpha1
+kind: ConstraintTemplate
+metadata:
+  name: gcp-gke-disable-legacy-endpoints-v1
+spec:
+  crd:
+    spec:
+      names:
+        kind: GCPGKEDisableLegacyEndpointsConstraintV1
+        plural: GCPGKEDisableLegacyEndpointsConstraintsV1
```

Comment:
let's user lowercase here per https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/#customresourcedefinition-v1beta1-apiextensions-k8s-io

(by default, `singular` is the lowercase form of `kind`, so we're skipping it)

Reply:
updated